Makefile

```diff
@@ -0,0 +1,242 @@
+	mv localnet/bisq-BTC_REGTEST_Bob_dao localnet/bob
+	# Remove the preconfigured bitcoin.conf in favor of explicitly
+	# parameterizing the invocation of bitcoind in the target below
+	rm -v localnet/bitcoind/bitcoin.conf
+	# Avoid spurious 'runCommand' errors in the bitcoind log when nc
+	# fails to bind to one of the listed block notification ports
+	echo exit 0 >> localnet/bitcoind/blocknotify
+
+# Deploy a complete localnet by running all required Bitcoin and Bisq
+# nodes, each in their own named screen window. If you are not a screen
+# user, you'll need to run each of the make commands manually in a
+# separate terminal or as a background job.
+#
+# NOTE: You MUST already be attached to a screen session for the
+# following commands to work properly.
+deploy: setup
+	screen -t bitcoin make bitcoind
+	sleep 2    # wait for bitcoind rpc server to start
+	make block # generate a block to ensure Bisq nodes get dao-synced
+	screen -t seednode make seednode
+	screen -t seednode2 make seednode2
+	screen -t alice make alice
+	screen -t bob make bob
+	screen -t mediator make mediator
+
+bitcoind: localnet
+	bitcoind \
+		-regtest \
+		-prune=0 \
+		-txindex=1 \
+		-server \
+		-rpcuser=bisqdao \
+		-rpcpassword=bsq \
+		-datadir=localnet/bitcoind \
+		-blocknotify='localnet/bitcoind/blocknotify %s'
+
+seednode: build
+	./bisq-seednode \
+		--baseCurrencyNetwork=BTC_REGTEST \
+		--useLocalhostForP2P=true \
+		--useDevPrivilegeKeys=true \
+		--fullDaoNode=true \
+		--rpcUser=bisqdao \
+		--rpcPassword=bsq \
+		--rpcBlockNotificationPort=5120 \
+		--nodePort=2002 \
+		--userDataDir=localnet \
+		--appName=seednode
+
+seednode2: build
+	./bisq-seednode \
+		--baseCurrencyNetwork=BTC_REGTEST \
+		--useLocalhostForP2P=true \
+		--useDevPrivilegeKeys=true \
+		--fullDaoNode=true \
+		--rpcUser=bisqdao \
+		--rpcPassword=bsq \
+		--rpcBlockNotificationPort=5121 \
+		--nodePort=3002 \
+		--userDataDir=localnet \
+		--appName=seednode2
+
+mediator: build
+	./bisq-desktop \
+		--baseCurrencyNetwork=BTC_REGTEST \
+		--useLocalhostForP2P=true \
+		--useDevPrivilegeKeys=true \
+		--nodePort=4444 \
+		--appDataDir=localnet/mediator \
+		--appName=Mediator
+
+alice: setup
+	./bisq-desktop \
+		--baseCurrencyNetwork=BTC_REGTEST \
+		--useLocalhostForP2P=true \
+		--useDevPrivilegeKeys=true \
+		--nodePort=5555 \
+		--fullDaoNode=true \
+		--rpcUser=bisqdao \
+		--rpcPassword=bsq \
+		--rpcBlockNotificationPort=5122 \
+		--genesisBlockHeight=111 \
+		--genesisTxId=30af0050040befd8af25068cc697e418e09c2d8ebd8d411d2240591b9ec203cf \
+		--appDataDir=localnet/alice \
+		--appName=Alice
+
+bob: setup
+	./bisq-desktop \
+		--baseCurrencyNetwork=BTC_REGTEST \
+		--useLocalhostForP2P=true \
+		--useDevPrivilegeKeys=true \
+		--nodePort=6666 \
+		--appDataDir=localnet/bob \
+		--appName=Bob
+
+# Generate a new block on your Bitcoin regtest network. Requires that
+# bitcoind is already running. See the `bitcoind` target above.
+block:
```

Comment:
I think this could use another sentence or so in the preamble. You end up needing to create blocks to test features like governance so helping new users fix common errors like "I created a proposal from Alice, but it isn't visible on Bob. Why not?" may help the onboarding.